From 1fff5fde3f5b2d167d5c51f9e7cf316fbd79886f Mon Sep 17 00:00:00 2001
From: Hebbar, Gururaja <gururaja.hebbar@ti.com>
Date: Tue, 8 Nov 2011 18:54:44 +0530
Subject: [PATCH 2/5] arm:omap:am335x: correct 32KHz clk rate

correct 32KHz clk (Divide down of PER PLL output) from 32678 to 32768

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/clock33xx_data.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/clock33xx_data.c b/arch/arm/mach-omap2/clock33xx_data.c
index 6763c59..f121196 100644
--- a/arch/arm/mach-omap2/clock33xx_data.c
+++ b/arch/arm/mach-omap2/clock33xx_data.c
@@ -44,7 +44,7 @@ void am33xx_init_timer_parent(struct clk *clk)
 
 static struct clk clk_32768_ck = {
 	.name		= "clk_32768_ck",
-	.rate		= 32678,
+	.rate		= 32768,
 	.ops		= &clkops_null,
 };
 
-- 
1.7.2.5

